aiida/repository/backend/disk_object_store.py

```diff
@@ -118,3 +120,116 @@ def get_object_hash(self, key: str) -> str:
         if self.container.hash_type != 'sha256':
             return super().get_object_hash(key)
         return key
+
+    def maintain(  # type: ignore # pylint: disable=arguments-differ,too-many-branches
+        self,
+        dry_run: bool = False,
+        live: bool = True,
+        override_pack_loose: bool = None,
+        override_do_repack: bool = None,
+        override_clean_storage: bool = None,
+        override_do_vacuum: bool = None,
+    ) -> dict:
+        """Performs maintenance operations.
+
+        :param full:
+            a flag to perform operations that require to stop using the maintained profile.
+        :param override_pack_loose:
```

Comment:
Is the `override_` prefix really necessary in the keyword itself? Seems more natural to me to just have it without. The docstring of `live` should then simply explain that it will automatically set values for these and the specific flags can be used to override, noting that for `live=True` not all of them may be overridden